src/python/ensembl/io/genomio/fasta/process.py

```diff
@@ -57,6 +60,7 @@ def prep_fasta_data(
     genbank_infile: Optional[PathLike],
     fasta_outfile: PathLike,
     peptide_mode: bool = False,
+    **kwargs: dict,
```

Comment:
Two different things about this line. First, for `*args` and `**kwargs` the type hint has to reflect the type of their content, not their type. In this case I would say `Any`:
```suggestion
    **kwargs: Any,
```
(remember to include it also in `from typing import Any, ...`)

Second, `dict` would only be supported in 3.9+ (they changed it to make it simpler). For 3.8 it would need to be `Dict` (and do its corresponding import as before).

Reply:
Yeah I don't like this solution, that's a bit too... Perlish

I'd rather we just write the parameters explicitly. That way we can add other parameters without breaking things (that's the point after all).

Reply:
changed to named params